server/chat-commands/admin.ts

```diff
@@ -885,8 +885,9 @@ export const commands: Chat.ChatCommands = {
 		const processes = new Map<string, ProcessData>();
 		const ramUnits = ["KiB", "MiB", "GiB", "TiB"];
 
+		const cwd = FS('.').path;
```

Comment:
Why do you sometimes use `FS('.').path` and sometimes `FS.ROOT_PATH`? They should both be the directory PS is in, right?

Reply:
Every FS('.').path reference is from before I added FS.ROOT_PATH. I'll swap the references.